lib/rbac/roles.rb

```diff
@@ -39,6 +39,15 @@ def delete(role)
       end
     end
 
+    def self.assigned_role?(role_name)
+      opts = { :limit => 100,
```

Comment:
@lindgrenj6 We can remove the limit since we are only expecting 1 item to come back since there can be only 1 role with that name.